### ISSUE TITLE
Parameter names should be in the lexiographical order

### ISSUE THREAD

**Issue by arai-a** [binast/binjs-ref#147]
moved from https://github.com/binast/ecmascript-binary-ast/issues/30

currently we're sorting the names in the alphabetical order, but it should be in lexiographical order per spec.
https://github.com/binast/binjs-ref/blob/e427f1b919ae0329514060079eed90e2deef544e/crates/binjs_es6/src/scopes.rs#L204

**Comment by Yoric** [comment]
Oh, right. Do you want to write a patch?

**Comment by arai-a** [comment]
yes, I'm writing patch for https://github.com/binast/ecmascript-binary-ast/pull/52 and it requires ordering,
so I'll fix this at the same time
